packages/core/src/types/model.ts

```diff
@@ -0,0 +1,18 @@
+import * as tf from '@tensorflow/tfjs-node-gpu';
+
+
+export interface PipcookModel {
```

Comment:
```suggestion
export interface UniModel {
```
Change to `UniModel`, too?

Reply:
done